.github/workflows/autoyapf.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           args: --style pep8 --recursive --in-place .
       - name: Check for modified files
         id: git-check
-        run: echo ::set-output name=modified::$(if git diff-index --quiet HEAD --; then echo "false"; else echo "true"; fi)
+        run: echo ::set-output name=modified::$(if git diff-index --quiet HEAD --; then echo "false"; fi)
       - name: Push changes
         if: steps.git-check.outputs.modified == 'true'
```

Comment:
Czy ta zmiana nie została już raz dodana w: https://github.com/mpfmorawski/git-to-know-me/pull/28/files
https://github.com/mpfmorawski/git-to-know-me/pull/28/files ?

Ustalcie z @KarolNiem który z was chce to dodać, bo inaczej będzie konflikt